consensus/istanbul/backend/handler.go

```diff
@@ -290,3 +298,181 @@ func (sb *Backend) UnregisterPeer(peer consensus.Peer, isProxiedPeer bool) {
 		}
 	}
 }
+
+// Handshake allows this node to identify itself to the peer as a validator and vice versa
+func (sb *Backend) Handshake(peer consensus.Peer) (bool, error) {
```

Comment:
IIUC, both peers will be sending the validatorproofmessage.  Is that necessary.  Can only the peer initiating the connection send it?

Reply:
Great point, you're totally right. I'll fix this

Reply:
https://github.com/celo-org/celo-blockchain/pull/873/commits/85644cb256f61dc4558dd503e4f0193497cc359a

Reply:
Fixed with: https://github.com/celo-org/celo-blockchain/compare/c3d137bd8cff0f965c41b1b4d4db738645149eec...806841c246c5127997c248fe9d8410ec0b1318b7?expand=1

Now the initiating peer only sends a message and the other peer just listens